tests/core/test_playback.py

```diff
@@ -1100,6 +1101,62 @@ def test(self):
         b.playback.change_track.assert_called_once_with(track2)
 
 
+class TesetCorePlaybackExportRestore(BaseTest):
+
+    def test_export(self):
+        tl_tracks = self.core.tracklist.get_tl_tracks()
+
+        self.core.playback.play(tl_tracks[1])
+        self.replay_events()
+
+        state = PlaybackState(
+            position=0, state='playing', tlid=tl_tracks[1].tlid)
+        value = self.core.playback._export_state()
+
+        self.assertEqual(state, value)
```

Comment:
I keep you hint in mind for future development and stay with the current implementation as (most) other tests are implemented in that way.